ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Jenkins_User/tasks/main.yml

```diff
@@ -41,7 +58,7 @@
     - adoptopenjdk
     # TODO: write a condition when NOT to run this step
     - skip_ansible_lint
-  when: (ansible_distribution != "MacOSX")
+  when: ansible_distribution != "MacOSX" and ansible_distribution != "Solaris"
```

Comment:
These conditionals are looking a bit odd. The comment says "for Redhat" but there seems nothing to guard that. If it's linux-specific perhaps we should be explicit about that instead of saying "Not blah" when something new shows up in the playbooks

Reply:
Agreed, I didn't want to break anything so the simplest solution was to skip Solaris. We should clarify if this is Rhel or Linux specific